lib/SymbolGraphGen/DeclarationFragmentPrinter.cpp

```diff
@@ -189,9 +189,17 @@ void DeclarationFragmentPrinter::printAbridgedType(const GenericTypeDecl *TD,
       case DeclKind::Protocol:
         printText(getTokenText(tok::kw_protocol));
         break;
-      case DeclKind::Class:
-        printText(getTokenText(tok::kw_class));
-        break;
+        case DeclKind::Class: {
+            auto *newthing = cast<ClassDecl>(TD);
+            if (newthing->isActor()) {
+                printText("actor");
+            } else if (newthing->isDistributedActor()) {
+                printText("distributed actor");
```

Comment:
+1 for adding `isExplicitActor` method here to force the semantic instead of depending the ordering.